src/v3/src/transformer/phone/phoneChallengeTransformer.ts

```diff
@@ -34,11 +34,12 @@ export const transformPhoneChallenge: IdxStepTransformer = ({ transaction, formB
   const resendStep = availableSteps?.find(({ name }) => name?.endsWith('resend'));
   if (resendStep) {
     const { name } = resendStep;
+    const smsMethodType = methodType === 'sms';
```

Comment:
There's some logic below on L50 that can use this as well, just need to invert it.